src/test/java/spoon/test/comment/CommentTest.java

```diff
@@ -110,12 +111,37 @@ public void testCombinedPackageInfoComment() {
 
 	@Test
 	public void testJavaDocComment() {
-		String EOL = System.getProperty("line.separator");
+		//the EOL is taken from JavaDocComment.java, which is commited in git with linux \n
+		//that is true on Windows too.
+		String EOL = "\n";
 
 		Factory f = getSpoonFactory();
 		CtClass<?> type = (CtClass<?>) f.Type().get(JavaDocComment.class);
 
 		CtJavaDoc classJavaDoc = (CtJavaDoc) type.getComments().get(0);
+		//contract: test that java doc is printed correctly
+		String str = classJavaDoc.toString();
+		StringTokenizer st = new StringTokenizer(str, "\n\r");
```

Comment:
Even on Windows there are situations when EOL is \n only
And even on Linux it can happen that you open file which contains \n\r as EOL.
In case of java sources checked out from repository it happens quite often